packages/big-design/src/components/Tree/TreeNode/TreeNode.tsx

```diff
@@ -39,16 +39,6 @@ const InternalTreeNode = <T,>({
   const isSelectable = value !== undefined && selectable?.type !== undefined && !isDisabled;
   const selectedChildrenCount = useSelectedChildrenCount({ selectedNodes: selectable?.selectedNodes, children });
 
-  useEffect(() => {
-    if (
-      focusable.focusedNode === id &&
-      nodeRef.current !== document.activeElement &&
-      document.activeElement !== document.body
-    ) {
-      nodeRef.current?.focus();
-    }
-  }, [focusable, id]);
```

Comment:
Sure, I was not aware it's for accessibility

Reply:
Hey folks, I have been trying to use this component inside a form and was having this weird issue of the table catching the focus from the entire page all the time, nice that you folks are tracking this.

Do you have any news regarding this? It's an important feature for our business since we are building a product details page to integrate with BigDesign. Thanks in advance!

Reply:
@felipesoares6 We are currently working on it